jest.config.js

```diff
@@ -6,4 +6,5 @@ module.exports = {
   testResultsProcessor: 'jest-sonar-reporter',
   collectCoverage: true,
   coveragePathIgnorePatterns: ['/node_modules/'],
+  testTimeout: 30000,
```

Comment:
Delete this line

Reply:
I believe extending timeout test crashing, can optimize running tests if lower network connection or weak hardware, but it does not harm test running performance on any hardware